WordPress/WordPressNotificationServiceExtension/Sources/NotificationService.swift

```diff
@@ -44,26 +48,22 @@ class NotificationService: UNNotificationServiceExtension {
 
             if let remoteNotifications = notifications,
                 remoteNotifications.count == 1,
-                let notification = remoteNotifications.first {
-
-                let identifier = notification.notificationId
-                if
-                    let title = notification.title,
-                    let type = notification.type
-                {
-                    notificationContent.title = "\(title) (\(identifier)) | \(type)"
-                }
-
-                if let subtitle = notification.timestamp {
-                    notificationContent.subtitle = subtitle
-                }
-
-                if let body = notification.url {
-                    notificationContent.body = body
-                }
-
-                if let metadata = notification.meta {
-                    notificationContent.userInfo["meta"] = metadata
+                let notification = remoteNotifications.first,
+                notification.kind == .comment,
```

Comment:
I wonder if we should check the notification type in the earlier `guard` statement in here, rather than assuming it is a comment. Once my latest changes roll out to the push server (hopefully end of this week), we'll probably be able to check `note_type` in the payload, which will contain a string such as `comment`.